source/QtCoralNetToolboxWidget.py

```diff
@@ -853,13 +857,29 @@ def taglabImport(self):
 
         """
         try:
-            # Import the points
-            self.parent().activeviewer.annotations.importCoralNetCSVAnn(self.predictions_file,
-                                                                        self.parent().project,
-                                                                        self.parent().activeviewer.image)
+            # Import the points, with project labels and active_image
```

Comment:
@gaiapavoni @massimomartinelli 

Unsure about getting the point's AI confidence scores to show up in the Panel correctly...